version/validate_compatibility.go

```diff
@@ -24,35 +24,36 @@ func ValidateCompatibility(client *houston.Client, out io.Writer, cliVer string,
 	}
 	// Skip check if AppConfig is nil or is cv is empty
 	if serverCfg != nil && cliVer != "" {
-		return compareVersions(serverCfg.Version, cliVer, out)
+		return CompareVersions(serverCfg.Version, cliVer, out)
 	}
 
 	return nil
 }
 
-func compareVersions(serverVer string, cliVer string, out io.Writer) error {
-	semVerServer, serverErr := parseVersion(serverVer)
-	if serverErr != nil {
-		return serverErr
+// CompareVersions print warning message if astro-cli has a variation in the minor version.  Errors if major version is behind.
+func CompareVersions(compareVer string, currentVer string, out io.Writer) error {
```

Comment:
better to make it back lowercase,

Reply:
It's OK as lowercase even now that it's used within two files? I thought the public functions were uppercase in go? I can switch it back to lower case if this isn't a public function.

Reply:
It's switched back to lowercase.